sphinx/ext/napoleon/__init__.py

```diff
@@ -44,6 +44,7 @@ class Config:
         napoleon_preprocess_types = False
         napoleon_type_aliases = None
         napoleon_custom_sections = None
+        napoleon_attr_annotations = False
```

Comment:
True by default?

Reply:
Oops, thanks for catching that ! Fixed it